ts/components/conversation/GroupNotification.tsx

```diff
@@ -75,35 +75,25 @@ export class GroupNotification extends React.Component<Props> {
           throw new Error('Group update is missing contacts');
         }
 
-        if (contacts.length === 1) {
-          if (contactsIncludesMe) {
-            return <Intl i18n={i18n} id="youJoinedTheGroup" />;
-          } else {
-            return (
+        const otherPeopleNotifMsg =
+          otherPeople.length === 1
+            ? 'joinedTheGroup'
+            : 'multipleJoinedTheGroup';
+
+        return (
+          <>
+            {otherPeople.length > 0 && (
               <Intl
                 i18n={i18n}
-                id="joinedTheGroup"
+                id={otherPeopleNotifMsg}
                 components={[otherPeopleWithCommas]}
               />
-            );
-          }
-        }
-
-        const joinedKey =
-          contacts.length > 1 ? 'multipleJoinedTheGroup' : 'joinedTheGroup';
```

Comment:
Next time you might consider making a minimal diff: on this line `contacts` should be `otherPeople`.

It would make everything easier.